extensions/storage-image-processing-api/functions/src/operations/output.ts

```diff
@@ -56,7 +242,83 @@ const structWebp = superstruct.object({
 const structTiff = superstruct.object({
   operation: superstruct.literal(name),
   format: superstruct.literal('tiff'),
-  // TODO options
+
+  /**
+   * quality, integer 1-100 (optional, default 80)
+   */
+  quality: superstruct.optional(
+    utils.coerceStringToInt(superstruct.size(superstruct.integer(), 1, 100)),
+  ),
+
+  /**
+   * Force TIFF output, otherwise attempt to use input format (optional, default true)
+   */
+  force: superstruct.optional(utils.coerceStringToBoolean()),
+
+  /**
+   * Compression options: lzw, deflate, jpeg, ccittfax4 (optional, default 'jpeg')
+   */
+  compression: superstruct.optional(superstruct.string()),
```

Comment:
```suggestion
  compression: superstruct.optional(superstruct.enum(['lzw', 'deflate', 'jpeg', 'ccittfax4'])),
```

Enum of possible values